input/projects/why-join-wip.md

```diff
@@ -0,0 +1,51 @@
+# Why should my project join the .NET Foundation?
+
+The .NET Foundation offers administrative and technical services to open source projects to help maintainers govern and develop their projects.
+
+.NET Foundation services aim to increase project stability and maturity and don't replace developers on member projects. Day-to-day development remains in the hands of project maintainers and contributors.
+
+Project benefits include:
+
+## Project guidance and mentoring
+
+New projects joining us will get mentorship on how to best run an open source community along with access to developers currently seeking .NET projects. We work with project leads to help ensure the project grows into a vibrant and welcoming community. And, of course, we love to share best practices for managing a friendly, collaborative, and innovative open source workplace, wherever you may be.
+
+## IP and legal
+
+Legal can be tricky. We can help. Projects in our organization will get assistance to organize and manage their IP assignments as well as infrastructure for tracking common, simple, Contribution License Agreements across all .NET Foundation projects. We can also help ensure that your collaborators respect the open source licenses and trademarks of projects assigned to the .NET Foundation. Need advice on legal issues? We offer support there as well.
```

Comment:
I don't think examples are a good fit for a summary page like this.

Reply:
I didn't suggest the particular project example, rather giving the range of services. I'm in the .NET Foundation for 2 years and don't know much about that, so my guess is that this will be also enigmatic for new people.

Reply:
The paragraph already gives examples. Would it help you if they were in a bullet point list? e.g.

> Legal can be tricky. We can help. Projects in our organization will get:
> * Assistance to organize and manage their IP assignments
> * Infrastructure for tracking common, simple, Contribution License Agreements across all .NET Foundation projects.
> * Help to ensure that your collaborators respect the open source licenses and trademarks of projects assigned to the .NET Foundation.
> * Support and advice on legal issues